schedule/yast/maintenance/autoyast_create_hdd_maintenance_migration.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: autoyast_create_hdd_maintenance_migration
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/wicked
+  - autoyast/repos
+  - autoyast/logs
+  - autoyast/autoyast_reboot
+  - installation/grub_test
+  - installation/first_boot
+  - console/system_prepare
+  - qa_automation/patch_and_reboot
+  - console/hostname
+  # - console/scc_cleanup_reregister
```

Comment:
Do we need keep those commented test modules?

Reply:
removed.